WindowsServerDocs/identity/ad-fs/operations/Configure-intranet-forms-based-authentication-for-devices-that-do-not-support-WIA.md

```diff
@@ -55,11 +55,11 @@ You can add Chrome or other user agents to the AD FS configuration that supports
 
 In AD FS configuration, add a user agent string for Chrome on Windows-based platforms:
 
-    Set-AdfsProperties -WIASupportedUserAgents ((Get-ADFSProperties | Select -ExpandProperty WIASupportedUserAgents) + "Mozilla/5.0 (Windows NT")
+    Set-AdfsProperties -WIASupportedUserAgents ((Get-ADFSProperties | Select -ExpandProperty WIASupportedUserAgents) + "Mozilla/5.0 (Windows NT"))
```

Comment:
Based on your constructive feedback comment, it appears we need to place the quote character correctly here:
```suggestion
    Set-AdfsProperties -WIASupportedUserAgents ((Get-ADFSProperties | Select -ExpandProperty WIASupportedUserAgents) + "Mozilla/5.0 (Windows NT)")
```
Reflecting on how parentheses work within general coding, I presume that both pairs of parentheses, as well as the quotes, should be placed symmetrically in relation to their encapsulated text strings.